src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -1353,6 +1355,22 @@ abstract class ManagedStrategy(
         }
     }
 
+    private fun processIntrinsicMethodEffectsOnObjectTracker(
```

Comment:
Maybe just `processIntrinsicMethodEffects` ?
In general, an intrinsic method can have other effects, affecting not only `objectTracker`.